openstack/cerebro/values.yaml

```diff
@@ -4,5 +4,5 @@ owner-info:
     - Alan Sergeant
     - Oliver Frendo
   helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/openstack/cerebro
-technicalUserPerRegion: null
+technicalUserPerRegion: TCC_CEREBRO_001
```

Comment:
You have set another user in the regional values for cerebro, who would overwrite this setting currently. Do you only want to configure it here and the regional settings should go?